### PR TITLE
DDF-2826 Increase default JVM memory to 4096MB

### PR DESCRIPTION
**Pull request by clockard**
#### What does this PR do?
Increase default JVM memory to 4096MB
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged)?
@ricklarsen @emmberk @mcalcote 
#### Select at least one member from relevant component team(s) from below (at least one component team member needs to approve the PR).

[Docs](https://github.com/orgs/codice/teams/docs)

#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@coyotesqrl
@figliold

#### What are the relevant tickets?
[DDF-2826](https://codice.atlassian.net/browse/DDF-2826)
#### Screenshots (if appropriate)
screen shot 2017-02-22 at 4 46 11 pm
screen shot 2017-02-22 at 4 46 31 pm
screen shot 2017-02-22 at 4 47 10 pm
screen shot 2017-02-22 at 4 47 35 pm

#### Checklist:
- [X] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc [hunk 1]
```diff
@@ -18,6 +18,10 @@ Adding other components does not have any security/hardening implications.
 
 These are the system/environment requirements to configure _prior_ to an installation.
 
+.Hardware Requirements
+* At least 4096MB of memory for ${branding}.
+** This amount can be increased to support memory intensive applications. See <<jvm-memory-configuration, Memory Considerations>>
```
Comment (emmberk) [inline distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc]: "memory intensive" -> "memory-intensive"

Comment (emmberk) [inline distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc]: Add period after `See <<jvm-memory-configuration, Memory Considerations>>`

File: distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc [hunk 2]
```diff
@@ -1,11 +1,30 @@
 
 Follow the below steps to start and stop ${branding}.
 
+.Memory Considerations
+[NOTE]
+====
+[[jvm-memory-configuration]]
+If the ${branding} will be running memory intensive applications, consider increasing the java memory. The java memory can be changed by updating the setenv script:
```
Comment (emmberk) [inline distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc]: "memory intensive" -> "memory-intensive"

File: distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor-table-contents.adoc [hunk 3]
```diff
@@ -16,12 +16,19 @@
 |false
```
Comment (emmberk) [inline distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor-table-contents.adoc]: Add a period after "Specifies the directory to be monitored"

File: distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor-table-contents.adoc [hunk 4]
```diff
@@ -16,12 +16,19 @@
...
+|Specifies the maximum number of concurrent files to be processed within a directory (maximum of 8).  If this number exceeds 8, 8 will be used in order to preserve system resources. Make sure that your system has enough memory to support the number of concurrent processing threads across all directory monitors.
+|1
+|true
+
+|ReadLock Time Interval
+|readLockIntervalMilliseconds
+|Integer
+|Specifies the time to wait (in milliseconds) before acquiring a lock on a file in the monitored directory.  This interval is used for sleeping between attempts to acquire the read lock on a file to be ingested.  The default value of 100 milliseconds is recommended.
+|100
+|true
```
Comment (emmberk) [inline distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor-table-contents.adoc]: Should details for [`processingMechanism`](https://github.com/codice/ddf/blob/7c702efa12c64098cefebcc244440a55044c02b6/catalog/core/catalog-core-directorymonitor/src/main/resources/OSGI-INF/metatype/metatype.xml#L32-L39) be added to this table?

File: distribution/docs/src/main/resources/_contents/_quickstart/quickstart-installing-contents.adoc [hunk 5]
```diff
@@ -17,6 +17,11 @@ These steps will demonstrate:
 
 These are the basic requirements to set up the environment to run a ${branding}.
 
+==== Hardware Requirements (Quick Install)
+
+* At least 4096MB of memory for ${branding}.
+** This amount can be increased to support memory intensive applications. See <<jvm-memory-configuration, Memory Considerations>>
```
Comment (emmberk) [inline distribution/docs/src/main/resources/_contents/_quickstart/quickstart-installing-contents.adoc]: Add period after See <<jvm-memory-configuration, Memory Considerations>>

File: distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc (hunk 1 above)
Comment (clockard) [inline distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc]: Fixed

File: distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor-table-contents.adoc (hunk 3 above)
Comment (clockard) [inline distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor-table-contents.adoc]: Fixed

File: distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc (hunk 2 above)
Comment (clockard) [inline distribution/docs/src/main/resources/_contents/_running/starting-intro-contents.adoc]: Fixed

File: distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor-table-contents.adoc (hunk 4 above)
Comment (clockard) [inline distribution/docs/src/main/resources/_contents/_tables/org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor-table-contents.adoc]: Yes, good catch! Fixed

File: distribution/docs/src/main/resources/_contents/_quickstart/quickstart-installing-contents.adoc (hunk 5 above)
Comment (clockard) [inline distribution/docs/src/main/resources/_contents/_quickstart/quickstart-installing-contents.adoc]: Fixed

File: distribution/ddf-common/src/main/resources/bin/setenv [hunk 6]
```diff
@@ -59,7 +59,7 @@ export EXTRA_JAVA_OPTS
 #
 
 # export JAVA_MIN_MEM # Minimum memory for the JVM
-export JAVA_MAX_MEM=2048M 
+export JAVA_MAX_MEM=4096M
```
Comment (coyotesqrl) [inline distribution/ddf-common/src/main/resources/bin/setenv]: Just a heads up, you could also set this as `4G` to be more human-readable.

**Pull request merged**